x_min: 0.3
x_max: 0.6
x_step: 0.05
y_min: -0.2
y_max: 0.2
y_step: 0.05
z_fixed: 0.2
qx: 0.0
qy: 1.0
qz: 0.0
qw: 0.0
samples_per_point: 50
wait_time: 0.5
output_file: "scan_data.csv"
